[core]/es_extended/client/functions.lua

```diff
@@ -442,18 +423,19 @@ end
 
 function ESX.Game.GetPlayers(onlyOtherPlayers, returnKeyValue, returnPeds)
     local players, myPlayer = {}, PlayerId()
+    local active = GetActivePlayers() 
 
-    for _, player in ipairs(GetActivePlayers()) do
-        local ped = GetPlayerPed(player)
+    for i=1, #active do 
+        local ped = GetPlayerPed(active[i]) 
```

Comment:
I think you should pass any variable active[i]. For example:  local currentPlayer = active[i] and use currentPlayer all usages.